Userland/Applications/DisplaySettings/MonitorSettingsWidget.cpp

```diff
@@ -214,10 +214,10 @@ ErrorOr<void> MonitorSettingsWidget::selected_screen_index_or_resolution_changed
         }
     }
 
-    auto dpi_label_value = String::formatted("{} dpi", screen_dpi.value());
-    if (screen_dpi.has_value() && !dpi_label_value.is_error()) {
+    if (screen_dpi.has_value()) {
+        auto dpi_label_value = TRY(String::formatted("{} dpi", screen_dpi.value()));
         m_dpi_label->set_tooltip(screen_dpi_tooltip.to_deprecated_string());
-        m_dpi_label->set_text(dpi_label_value.release_value());
+        m_dpi_label->set_text(dpi_label_value);
```

Comment:
Let's do:
```c++
m_dpi_label->set_text(move(dpi_label_value));
```
To avoid needlessly copying the `String` object (which is just a temporary ref-count change, but can still be avoided)

Reply:
sounds good. I've added this changes.